src/tests/editUpdateClear.test.js

```diff
@@ -0,0 +1,46 @@
+import addTask from '../modules/addTask.js';
+import taskArray from '../modules/taskArray.js';
+
+document.body.innerHTML = `<div><input type="text" class="input" id="add-input" placeholder="Add to your list...">
+<div class="list-container">
+<ul class="list"></ul>
+</div></div>`;
+
+describe('edit function', () => {
+  test('should edit the task description', () => {
+    addTask('Hello Typescript');
+    const input = document.querySelector('.task-label');
+    input.value = 'Bye Vanilla JS';
+    input.dispatchEvent(new Event('input'));
+    expect(document.querySelector('.task-label').value).toBe('Bye Vanilla JS');
+  });
+  test('should update the taskArray', () => {
+    const input = document.querySelector('.task-label');
+    input.value = 'Hello Typescript';
+    input.dispatchEvent(new Event('input'));
+    expect(taskArray).toEqual([{
+      description: 'Hello Typescript',
+      completed: false,
+      index: 1,
+    }]);
+  });
+});
+
+describe('clear function', () => {
+  test('should clear the task description', () => {
+    const input = document.querySelector('.task-label');
+    input.value = '';
+    input.dispatchEvent(new Event('input'));
+    expect(document.querySelector('.task-label').value).toBe('');
+  });
+  test('should update the taskArray', () => {
+    const input = document.querySelector('.task-label');
+    input.value = '';
+    input.dispatchEvent(new Event('input'));
+    expect(taskArray).toEqual([{
+      description: '',
+      completed: false,
+      index: 1,
+    }]);
+  });
+});
```

Comment:
![image](https://github.com/SergioPeralta22/To-Do-List-Webpack/assets/93039359/b000563f-df61-4d11-8b67-3064c86279ee)

- [ ] Great work on implementing the tests so far 👏 but I guess you misunderstood the requirements because if you see the requirements above then there is clearly mentioned that you are supposed to test those `functions` which are responsible for checkbox and clearing the completed task so according to these requirements only two functions should be tested which are in your checker.js file which is **check()** and **clearCompleted()**  so kindly test these methods to follow the project requirements and to make sure that your app functionality is working as expected 😊👍